pipelines/rj_smtr/projeto_subsidio_sppo/tasks.py

```diff
@@ -104,24 +104,27 @@ def get_raw(
                 # busca acoes do recurso
                 for action in recurso["actions"]:
                     row = {
+                        "data_recurso": a["createdDate"],
```

Comment:
@gmartinsoc aqui só nao existe `a`
```suggestion
                        "data_recurso": action["createdDate"],
```